### DISCUSSION TITLE
How to learn/use Rectangle Pro

### DISCUSSION THREAD

**Discussion by listors** [rxhanson/RectanglePro-Community#422] [General]
I bought Pro because I like Rectangle that I have used for a while. 
There is one video available. I understand that it is time consuming to make video.
There are a few Show and tell. They are helpful. 
I have scrolled the discussions. Not so effective nor focused.

So this boils down to a question: 
-  How to learn effectively the different features and how to combine them? Is there any manual? Or how to go about it?

**Comment by listors** [comment]
Here're a couple of additional questions:
- How can I invoke a set of apps to be opened within its window of a defined layout?
- How can I open a new Safari window and move it to a new Space?
- How can I open several Safari windows in separate Spaces? (maybe not a Rectangle solution)

Thanks
Best regards,
Leif

**Comment by rxhanson** [comment]
Thanks for purchasing. 

> How to learn effectively the different features and how to combine them? Is there any manual? Or how to go about it?

Maybe at the very least, a Show & Tell for each tab of the settings window would be helpful?

> How can I invoke a set of apps to be opened within its window of a defined layout?

I'm guessing you've already looked at https://github.com/rxhanson/RectanglePro-Community/discussions/111

To execute an app layout, you can add a keyboard shortcut to layout as shown in that discussion. You can also execute layouts from the menu bar menu, the window throw, or the snap panel. Let me know if this is still not clear.

> How can I open a new Safari window and move it to a new Space?
> How can I open several Safari windows in separate Spaces? (maybe not a Rectangle solution)

There's no public API for spaces, so move the to next/prev Space keyboard shortcut is the only Spaces operation that is included at this time.

**Reply by pinkgalaxy** [reply]
Just a quick few seconds snippet preview like the Mac trackpad demonstrations would be helpful!

**Comment by listors** [comment]
Thanks for your kind and helping reply. I'll dive further into the Rectangle Pro world.

**Comment by pinkgalaxy** [comment]
> I bought Pro because I like Rectangle that I have used for a while. There is one video available. I understand that it is time consuming to make video. There are a few Show and tell. They are helpful. I have scrolled the discussions. Not so effective nor focused.
> 
> So this boils down to a question:
> 
> * How to learn effectively the different features and how to combine them? Is there any manual? Or how to go about it?

MY THOUGHTS EXACTLY! I purchased Rectangle Pro so that I can be more efficient, but because I don't understand the terminology, my experience with it so far has been extremely frustrating and time-consuming trying to understand which each feature does, how they're different from one another, and navigate why/when to use which feature to optimize efficiency. I'm hoping we can get quick tutorial video snippets or even an FAQ page explaining with examples of the different features.

**Reply by rxhanson** [reply]
Sorry, it's definitely not my intent to cause frustration. As is always the case, time is my limiting factor, and I do intend to produce more tutorial content when I can.

**Reply by pinkgalaxy** [reply]
> rxhanson
lolol I would greatly appreciate it as a non-tech savvy person who's trying to keep up with the times. But in the meantime, here is my sad attempt at demo video for those who'd find it helpful:

SNAP PANEL:
https://github.com/rxhanson/RectanglePro-Community/assets/164553001/d10a648e-fc89-40f4-ba89-340584ac5a9f

**Comment by pinkgalaxy** [comment]
WINDOW THROW:
https://github.com/rxhanson/RectanglePro-Community/assets/164553001/91947692-6db6-40c9-bafc-e6889b1c3827

**Comment by pinkgalaxy** [comment]
STASH:

https://github.com/rxhanson/RectanglePro-Community/assets/164553001/9f27415a-6522-4213-95ba-f363f29891bd

**Reply by rxhanson** [reply]
Woah, thanks for making these! If you want to make some for any of my other apps, I'll give you some license keys :)

**Reply by pgitkn** [reply]
Agreed w/ the original thought - tutorials would be greatly appreciated.  Perhaps someone on Fivrr.com could knock out a few dozen tutorials for $100-200?   I'm certain I'm barely scratching the surface of what Rectangle, Charmstone, etc are capable of but I don't have the time necessary to figure it out without some guidance...  Thanks for my most-important-app!

**Comment by pinkgalaxy** [comment]
hahahaha hey, yo! glad you appreciate them!
I'll try out Charmstone, Superkey, and Multitouch +/- Keylimepie and get
back to you :)

**Comment by technocrat** [comment]
Without hello world, it's goodbye app. 

I've got two windows open and I want to tile them upper left and upper right. I got as far as pinning BBEdit on the right. How to put Warp on the left, I don't know. I'm not going to spend an hour trying to figure out how to make a $10 app work.

I'm not going to leave a trash review, but if I were I'd split it with ***** for features and * for usability

**Reply by rxhanson** [reply]
Thanks for the feedback. 

There are a few ways to place a window in the top left or top right quarter of the screen:
1. Drag a window until your cursor reaches the top left or right corner of the screen and then release.
2. Press control+option+u to move the frontmost window to the top left or control+option+i to move it to the top right.
3. With your cursor over the window you want to move, hold control+command and move the cursor slightly up & left or up & right, then release the keys.

> I'm not going to leave a trash review, but if I were I'd split it with ***** for features and * for usability

I'm not sure what this means. Which features do you dislike, and what makes it so hard to use? (I'm not trying to be rude, just trying to understand where to improve or what really needs documentation).

**Comment by technocrat** [comment]
I understand what it is to be so familiar with a piece of software works to find it hard to understand why someone else can’t quite trip to how it operates. (That’s a great justification for why I have published only one piece of software since I compiled my first FORTRAN deck 59 years ago.)Besides being a Boomer of advanced years, I don’t do videos just as I don’t do Windows. So maybe I’m not the target audience for this app. And I tend to be pretty harsh with software that doesn’t just work. I’ve tried and discarded hundreds (maybe thousands) of programs that didn’t just work out of the box under various iterations of JCL, DOS/pre-DOS, proprietary standalone word processors, Windows through the third BSOD in XP, Unix v7, Irix, macOS from Tiger and most versions of Ubuntu. It doesn’t have to be simple, but there does have to be an entry point that’s either intuitive or clearly shown. I’ll try options 2 and 3 (1 didn’t work)Thanks for getting back to me.Richard On Apr 13, 2024, at 9:31 PM, Ryan Hanson ***@***.***> wrote:﻿
Thanks for the feedback.
There are a few ways to place a window in the top left or top right quarter of the screen:

Drag a window until your cursor reaches the top left or right corner of the screen and then release.
Press control+option+u to move the frontmost window to the top left or control+option+i to move it to the top right.
With your cursor over the window you want to move, hold control+command and move the cursor slightly up & left or up & right, then release the keys.


I'm not going to leave a trash review, but if I were I'd split it with ***** for features and * for usability

I'm not sure what this means. Which features do you dislike, and what makes it so hard to use? (I'm not trying to be rude, just trying to understand where to improve or what really needs documentation).
